toolkit/pythonit_toolkit/pastaporto/tokens.py

```diff
@@ -24,3 +25,18 @@ def decode_service_to_service_token(
         algorithms=["HS256"],
         options={"require": ["exp", "iss", "aud", "iat"]},
     )
+
+
+def generate_service_to_service_token(secret, issuer: str, audience: str):
+    if not secret:
+        raise ValueError("Secret can not be empty")
+
+    return jwt.encode(
+        {
+            "iss": issuer,
+            "aud": audience,
+            "exp": datetime.now(tz=timezone.utc) + timedelta(seconds=30),
+        },
+        secret,
```

Comment:
```suggestion
        str(secret),
```

it is not needed, but makes our life easier when in starlette we use `Config#Secret`